src/main/java/ezschedule/commons/core/index/Index.java

```diff
@@ -22,6 +22,9 @@ private Index(int zeroBasedIndex) {
 
         this.zeroBasedIndex = zeroBasedIndex;
     }
+    public static Index initIndex(int zeroBasedIndex) {
+        return new Index(zeroBasedIndex);
+    }
```

Comment:
This seem redundant, and might also cause confusion. I recommend using with one of the following method found below: `fromZeroBased` or `fromOneBased`